src/gui/tray/NotificationCache.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <set>
+#include <QObject>
+
+namespace OCC {
+
+struct Notification
```

Comment:
@FlexW I guess this struct should be nested into `NotificationCache` just to avoid possible namespace collisions in the future. Also, it feels like this struct is a part of `NotificationCache`.